libs/java/service/src/main/java/ca/bc/gov/tno/services/net/JsonRestTemplate.java

```diff
@@ -0,0 +1,60 @@
+package ca.bc.gov.tno.services.net;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import org.springframework.http.client.ClientHttpRequestFactory;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.json.MappingJackson2HttpMessageConverter;
+import org.springframework.web.client.RestTemplate;
+
+public class JsonRestTemplate extends RestTemplate {
```

Comment:
Custom `RestTemplate` that will actually throw the real exception when it's unable to deserialize.  However I'm 100% sure this will break in other areas.  But at least we'll get an error that'll help debug.